packages/react/src/components/InAppMessaging/BannerMessage/BannerMessage.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Heading,
+  Icon,
+  Image,
+  Text,
+  View,
+  Button,
+  Flex,
+} from '@aws-amplify/ui-react';
```

Comment:
We should be able to import these from the `primitives` folder, additionally we want to avoid importing all the primitives to help with tree shaking. Example (not guaranteeing my syntax is correct):
```
import { Flex } from '../../../primitives/Flex';
import { Heading } from '../../../primitives/Heading';
// etc
```